.github/workflows/infrastructure.yaml

```diff
@@ -22,26 +21,22 @@ permissions:
   pull-requests: write
 
 jobs:
-  terraform:
+  plan:
+    if: github.event_name == 'pull_request'
     strategy:
       fail-fast: false
       matrix:
-        config:
-          - environment: dev
-            account_id: <DEV_ACCOUNT_ID>
-            tf_deploy_override: ${{ github.event_name == 'workflow_dispatch' && true || false }} # Allow manual deployment of Dev
-          # - environment: prod
-          #   account_id: <PROD_ACCOUNT_ID>
-          #   tf_deploy_override: false
+        environment: <MATRIX>
+    uses: tx-pts-dai/github-workflows/.github/workflows/tf-plan.yaml@v0.16.0
```

Comment:
```suggestion
    uses: tx-pts-dai/github-workflows/.github/workflows/tf-plan.yaml@v0.18.0
```

Reply:
Minor... Or should we let tam-cli update the workflow version ?

Reply:
Or maybe renovate 🤔  ?